code/datums/quirks/positive_quirks.dm

```diff
@@ -207,6 +207,85 @@
 	// We've either added or removed TRAIT_NIGHT_VISION before calling this proc. Just refresh the eyes.
 	eyes.refresh()
 
+/datum/quirk/bilingual
+	name = "Bilingual"
+	desc = "Over the years you've picked up an extra language!"
+	icon = FA_ICON_GLOBE
+	value = 4
+	gain_text = span_notice("Some of the words of the people around you certainly aren't common. Good thing you studied for this.")
+	lose_text = span_notice("You seem to have forgotten your second language.")
+	medical_record_text = "Patient speaks multiple languages. Showoff."
+	var/list/possible_languages = list(
+		/datum/language/aphasia,
+		/datum/language/beachbum,
+		/datum/language/calcic,
+		/datum/language/draconic,
+		/datum/language/drone,
+		/datum/language/moffic,
+		/datum/language/monkey,
+		/datum/language/mushroom,
+		/datum/language/nekomimetic,
+		/datum/language/piratespeak,
+		/datum/language/shadowtongue,
+		/datum/language/slime,
+		/datum/language/sylvan,
+		/datum/language/terrum,
+		/datum/language/uncommon,
+		/datum/language/voltaic,
+		/datum/language/xenocommon
+	)
+	var/extra_language
+	mail_goodies = list(/obj/item/taperecorder, /obj/item/clothing/head/frenchberet, /obj/item/clothing/mask/fakemoustache/italian)
+
+/datum/quirk/bilingual/add(client/client_source)
+	var/mob/living/carbon/human/human_holder = quirk_holder
+	//prevents yourself from learning a language you already have
+	for(var/datum/language/spoken as anything in possible_languages)
+		if(human_holder.has_language(spoken))
+			possible_languages -= spoken
+	extra_language = pick(possible_languages)
+	human_holder.grant_language(extra_language)
+
+/datum/quirk/bilingual/remove()
+	var/mob/living/carbon/human/human_holder = quirk_holder
+	human_holder.remove_language(extra_language)
+
+/datum/quirk/mutated
+	name = "Mutated"
+	desc = "You've already had experience as a labrat for the Nanotrasen Genetic Project."
+	icon = FA_ICON_DNA
+	value = 8
+	gain_text = span_notice("You remember one of your genes was altered.")
+	lose_text = span_notice("You seem to have lost your mutation.")
+	medical_record_text = "Patient's DNA shows signs of genetic tampering."
+	var/selected_gene
+
+/datum/quirk/mutated/add(client/client_source)
+	var/mob/living/carbon/human/human_holder = quirk_holder
+	//picks a gene based on weight
+	selected_gene = pick_weight(list(
+		/datum/mutation/human/temperature_adaptation = 4,
+		/datum/mutation/human/antenna = 15,
```

Comment:
hey come on now, use initial(typepath.difficulty) so it updates when the var is changed by a later developer

Reply:
Guh, but isn't difficulty a higher number for the rarer it is?

Would it be more like something where it's:

`= 100/temperature_adaptation.difficulty`?

Reply:
hmm normalising a list of values from 0-100 based on another scale from 0-max difficulty, kinda painful.

maybe 100/difficulty will be enough

Reply:
I might need your expertise on this since I don't know how it'll pull the difficulty value from each specific mutation.

Reply:
```
picklist = list()
list(typepatha, typepathb)
for typepath in list:
    pickchance = 100/initial(typepath.difficulty)
    picklist[typepath] = difficulty
pick_from(picklist)
```
pseudocode obviously but should